src/plays/age-calculator-by-react/Calculator.jsx

```diff
@@ -0,0 +1,19 @@
+import Header from './Header';
```

Comment:
Import statements should be sorted alphabetically and each import should be on a new line to improve readability.

Place the `Header, Input, Output`, and `useAge` imports in the order of their appearance in the code.